src/geonovum/informative.js

```diff
@@ -0,0 +1,13 @@
+// Module geonovum/informative
```

Comment:
See my previous comment about us making this a core module instead. We can then add the localization for English and Dutch.

Reply:
Agreed.

Reply:
Regarding this module: how would you like to proceed? Should I for now keep this geonovum module and as soon as we have a core module, change it?

Reply:
ok, as part of this PR, let's update `core/informative.js`. Could you delete this file. Then we just need to copy the structure and test from:

https://github.com/w3c/respec/pull/2135/files

Reply:
Could you do the the changes to core/informative.js ? I'll do the test, as the test file doesn't exist yet.

Reply:
>  Should I for now keep this geonovum module and as soon as we have a core module, change it?

Let's fix things as we go. That way we eliminate duplication and reduce technical debt.

Reply:
Okay. And I'll do the changes to core/informative.js (better: give it a try )